### PR TITLE
feat: support output.chunkLoadingGlobal

### PR DESCRIPTION
**Pull request by 9aoy**
## Summary

[[Feature Request]: support output.chunkLoadingGlobal](https://github.com/web-infra-dev/rspack/issues/2388)

## Related issue (if exists)

Closes https://github.com/web-infra-dev/rspack/issues/2388

## Types of changes

- [ ] Docs change / Dependency upgrade
- [ ] Bug fix
- [x] New feature / Improvement
- [ ] Refactoring
- [ ] Breaking change

## Checklist

- [ ] I have added changeset via `pnpm run changeset`.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/rspack/tests/Defaults.unittest.ts [hunk 1]
```diff
@@ -259,6 +259,7 @@ describe("snapshots", () => {
 		  "output": {
 		    "assetModuleFilename": "[hash][ext][query]",
 		    "chunkFilename": "[name].js",
+		    "chunkLoadingGlobal": "webpackChunk@rspack/core",
```
Comment (underfin) [inline packages/rspack/tests/Defaults.unittest.ts:262]: Look like it is not a valid identifier. It need use `to_identifier` to normalize it.

**Comment by ahabhgk** [comment]
!eco-ci

**Comment by ahabhgk** [comment]
⏳ Triggered ecosystem CI: [Open](https://github.com//web-infra-dev/rspack-ecosystem-ci/actions/runs/4655374086)

File: packages/rspack/tests/Defaults.unittest.ts (hunk 1 above)
Comment (9aoy) [reply packages/rspack/tests/Defaults.unittest.ts:262]: ok. I add `to_identifier` in rust, so it looks no change in js.

**Review by underfin — APPROVED** [review]

**Pull request merged**